index.html

```diff
@@ -5,8 +5,57 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel="stylesheet" type="text/css" href="./styles/index.css"/>
 </head>
 <body>
+    <header>
+        <section class="header-cover" id="top">
+            <h1 class="title"> Kristin's Weather Report</h1>
+            <h3 id="subtitle">For the lovely city of ✨<span id="cityNameDisplay">Seattle</span>✨</h3>
+        </section>
+    </header>
+    <main>
+        <section class ="content-wrapper">
+            <div class="items-wrapper">
+                    <div class="item-wrapper" id="temperatureContainer">
```

Comment:
You might consider making the `temperatureContainer`, `cityNameContainer`, `skyContainer`, and `weatherGardenContainer` section elements